aws-logs-subscriptionfilter/example_inputs/inputs_1_create.json

```diff
@@ -0,0 +1,8 @@
+{
+    "desiredResourceState": {
+        "DestinationArn": "arn:aws:lambda:us-west-2:356761496836:function:TestLambdaFunc",
```

Comment:
Remove account information

Reply:
Will remove shortly. Need to create an template for dev resources to avoid hard-coding for quick testing.

Reply:
Same as above. Avoid hardcoding resources. Create a dependency stack that can output the required values.